### PR TITLE
[MINOR]EDA-1405 Redis and conversions

### PR DESCRIPTION
**Pull request by srobledo-eb**
Created files: redis.py, redis_constants.py, environment.py
Created methods 'set_game' and 'get_game' to save and retrieve a dictionary with the state of a game.
Each of the above methods make use of helping methods to convert a game into a dictionary and later to a JSON string (to save) as well as the opposite process (from JSON to Game for retrieving info)

### CODE REVIEW

**Review by lolivera-eb — APPROVED** [review]

File: game/game.py [hunk 1]
```diff
@@ -27,11 +27,19 @@
 
 class Game:
     def __init__(self, players_names: List[str]) -> None:
-        self.game_id = str(uuid.uuid1())
-        self.players = self.create_players(players_names)
-        self.current_player = self.players[0]
-        self.board = Board(self.players)
-        self.remaining_moves = TOTAL_MOVES
+        if len(players_names) > 0:
```
Comment (arielaranda-eb) [inline game/game.py:30]:
```
if players_names:
    ...
```
looks more pythonistic

File: game/board.py [hunk 2]
```diff
@@ -199,3 +198,12 @@ def wall_removal(self, wall):
...
             self.crosses[row][col] = EMPTY_SPOT
+
+    def get_pawns_positions(self):
+        pawns = {}
+        for row in range(0, DEFAULT_BOARD_SIZE):
+            for col in range(0, DEFAULT_BOARD_SIZE):
+                if self.board[row][col]:
+                    pawn = self.board[row][col]
+                    pawns[f'pawn_{pawn.side}'] = (row, col)
+        return pawns
```
Comment (arielaranda-eb) [inline game/board.py:209]: I think this works well... 
If you want... try to think of a way to do this without having to go all over the board.

**Review by arielaranda-eb — APPROVED** [review]
lgtm

File: game/game.py (hunk 1 above)
Comment (srobledo-eb) [reply game/game.py:30]: That's true. I'll fix it for next PR

Comment (juanvallejo-eb) [reply game/game.py:30]: +1

File: game/board.py (hunk 2 above)
Comment (srobledo-eb) [reply game/board.py:209]: I will consider modifications in future sprints in order to save the position of pawns in their own attrtbutes

**Review by juanvallejo-eb — APPROVED** [review]
lgtm

**Pull request merged**